### PR TITLE
Issue 8084: Decrease P3A upload interval.

### PR DESCRIPTION
**Pull request by iefremov**
## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [x] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [x] Linux
  - [ ] macOS
  - [ ] Windows
- [x] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [x] Ran `git rebase master` (if needed).
- [x] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [ ] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: components/p3a/brave_p3a_service.cc [hunk 1]
```diff
@@ -43,7 +43,7 @@ constexpr char kLastRotationTimeStampPref[] = "p3a.last_rotation_timestamp";
 
 constexpr char kDefaultUploadServerUrl[] = "https://p3a.brave.com/";
 
-constexpr uint64_t kDefaultUploadIntervalSeconds = 60 * 60;  // 1 hour.
+constexpr uint64_t kDefaultUploadIntervalSeconds = 60;  // 1 minute.
```
Comment (NejcZdovc) [inline components/p3a/brave_p3a_service.cc]: nit: maybe we could do `1 * base::Time::kSecondsPerMinute` so that it's clearer

Comment (riastradh-brave) [reply components/p3a/brave_p3a_service.cc]: We should transition all of the randomized delays from integral numbers of seconds to base::Time -- that's a more general issue that we should fix across the board.

Comment (iefremov) [reply components/p3a/brave_p3a_service.cc]: @riastradh-brave probably, `base::TimeDelta`, not `base::Time`? Is there an issue for this?

Comment (riastradh-brave) [reply components/p3a/brave_p3a_service.cc]: (Sorry, wires crossed -- base::time::kSecondsPerMinute is an integer number of seconds; using base::Time itself is a different issue.  Don't mind me!)

**Comment by iefremov** [comment]
With the help of @linhkikuchi manually verified that CI is ok - only one unrelated build failure on mac

**Pull request merged**